contracts/KyberProxy.sol

```diff
@@ -9,22 +10,25 @@ contract KyberProxy is TokenConverter, Ownable {
     
     uint256 private constant MAX_UINT = uint256(0) - 1;
 
-    KyberNetworkProxy public converter;
+    KyberNetworkProxy kyberProxy;
```

Comment:
Maybe this name can be a little misleading (KyberProxy and kyberProxy), the contracts are different things maintained by different devs. I would suggest rename kyberProxy to kyber